packages/flutter/lib/src/material/stepper.dart

```diff
@@ -0,0 +1,648 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'package:meta/meta.dart';
+
+import 'button.dart';
+import 'colors.dart';
+import 'debug.dart';
+import 'flat_button.dart';
+import 'icon.dart';
+import 'icons.dart';
+import 'ink_well.dart';
+import 'material.dart';
+import 'theme.dart';
+
+// TODO(dragostis): Missing functionality:
+//   * mobile horizontal mode w/ adding/removing steps
+//   * alternative labeling
+//   * stepper feedback in the case of high-latency interactions
+
+/// The state of a step which is used to control the style of the circle and
```

Comment:
Prefer "The state of a [Step]. This affects the appearance of the step's circle and text." or some such.